ci/test_wheel_dask_cudf.sh

```diff
@@ -23,9 +23,6 @@ manylinux="manylinux_${manylinux_version}"
 RAPIDS_PY_WHEEL_NAME="cudf_${manylinux}_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./local-cudf-dep
 python -m pip install --no-deps ./local-cudf-dep/cudf*.whl
 
-# Always install latest dask for testing
-python -m pip install git+https://github.com/dask/dask.git@main git+https://github.com/dask/distributed.git@main git+https://github.com/rapidsai/dask-cuda.git@branch-23.12
```

Comment:
@vyasr Is the removal of `dask-cuda` from here because we now publish nightly wheels per PR merge?

Reply:
Partially. Aside from that issue where we wouldn't always get the absolute latest commit of dask-cuda due to the nightly versioning change, the other relevant change in this PR is [the build_wheel.sh change that allows us to pick up nightlies when installing the pip package](https://github.com/rapidsai/cudf/pull/14364/files#diff-904c7127d3d586dca8c2bcf3055ec8acecc095a9e1d53ff81d94150698c0e784R39). We could have made that change before and it would have been sufficient the vast majority of the time.

Reply:
👍